.gitlab-ci.yml

```diff
@@ -0,0 +1,15 @@
+before_script:
+  - apt-get update
```

Comment:
cosmetic: to be consistent with the other commands, use `apt` here instead of `apt-get`